src/app/pages/auth/components/ory-form/ory-form.component.html

```diff
@@ -0,0 +1,19 @@
+<div class="form">
+  <div dir="auto" class="text text-justify">
+    {{ 'ENROL_REGISTRATION_ORY_DESC' | translate }}
+  </div>
+  <div class="qr-code">
+    <img src="assets/imgs/radar-qr-code.png" alt="RADAR QR code" />
+  </div>
+  <div class="ion-padding bottom" *ngIf="!loading">
+    <ion-button
+      ion-button
+      class="bt bt--full"
+      round
+      full
+      (click)="scanQRHandler()"
```

Comment:
update to use only one QR code scan button for both MP and SEP scanning, and then app can route to the correct login.